src/embit/descriptor/arguments.py

```diff
@@ -14,9 +14,11 @@ def __init__(self, fingerprint: bytes, derivation: list):
     @classmethod
     def from_string(cls, s: str):
         arr = s.split("/")
+        if len(arr[0]) % 2 != 0:
```

Comment:
I believe this check aims to avoid unhandled exceptions with `unhexlify`, right? Maybe `unhexlify` could be inside a `try/except` to also catch other invalid strings and non hex characters.
Also, remembering this lib is made to run on limited devices, I would reduce verbosity to a minimum:
"Invalid fingerprint format"
"Invalid fingerprint length"